### ISSUE TITLE
Hidden Dynamic Grid stacking when shown

### ISSUE THREAD

**Issue by mattivie14** [uikit/uikit#1348]
I have a hidden div and it has a dynamic grid inside it (it also has the filtering option setup if that matters), when the js triggers that div to show, all the panels in the dynamic grid stack on top of eachother. I tried to put "data-uk-check-display" and "data-uk-observe" in different places and I couldnt get either of those to fix it. How do I fix this?

**Comment by malles** [comment]
You don't need to place the `data-uk-check-display` tag on the grid, but the event needs to be fired. The grid reacts automatically to it. Run `UIkit.trigger('display.uk.check');` after you show the div with js.

**Comment by mattivie14** [comment]
That doesnt seem to do anything, I don't get any errors in the console, and even if I run that line directly in the console it doesnt fix the grid.

**Comment by malles** [comment]
Maybe try `UIkit.$html.trigger('display.uk.check')`

**Comment by mattivie14** [comment]
That also doesnt seem to work.
Am I missing some js/css file that I need? this is what I have on that page now:

    <link rel="stylesheet" href="../css/uikit.gradient.css" />
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
    <script src="../js/uikit.min.js"></script>
    <script src="../js/components/grid.js"></script>
    <script src="../js/core/grid.js"></script>

Also if I click the filter options above the dynamic grid, the grid then starts working fine, if that helps at all.

**Comment by malles** [comment]
You can lose the `../js/core/grid.js` file, it's compiled into `ukit.min.js`. It should be fine.
Can you create a codepen/jsfiddle?

**Comment by malles** [comment]
Okay, another go: try `UIkit.$html.trigger('changed.uk.dom');`

**Comment by mattivie14** [comment]
Thanks, that one worked!
